clang/include/clang/Driver/Options.td

```diff
@@ -4638,10 +4638,14 @@ def image__base : Separate<["-"], "image_base">;
 def include_ : JoinedOrSeparate<["-", "--"], "include">, Group<clang_i_Group>, EnumName<"include">,
     MetaVarName<"<file>">, HelpText<"Include file before parsing">,
     Visibility<[ClangOption, CC1Option]>;
-def include_footer : Separate<["-"], "include-footer">, Group<clang_i_Group>,
+def include_internal_footer : Separate<["-"], "include-internal-footer">, Group<clang_i_Group>,
```

Comment:
Is there a possibility where we want to include header and not footer...or the other way around? Just wondering if we need two options here.

Thanks

Reply:
Good question. @premanandrao What do you think?

Reply:
The behavior of `-include-internal-header` and `-include-internal-footer` is more than just the ability to pass in a file to be included, it also designates where in the preprocessing the file is expanded, so we need to have 2 options.

Reply:
Thanks Mike!